emails/_newsletter-0524.tsx

```diff
@@ -0,0 +1,153 @@
+            May 29th 5:30–8PM @ Twitch HQ
+          </Text>
+        </Link>
+        <div className="mb-8 mt-4 sm:px-4">
+          <Img
+            className="w-full"
+            src="https://tj8xrxsxqdtxeknk.public.blob.vercel-storage.com/email/pasifika%20orgs-9izCj4uKUVq7xr1It0nwZF0L17abk0.png"
+          />
+        </div>
+        <div className="rounded bg-stone-100 p-2 text-center sm:p-4">
+          <Text className="mb-2 mt-0 text-xs font-bold">
+            Then,{" "}
+            <Link href="https://lu.ma/ofzvwvaj" className="text-brown-600">
+              share the RSVP
+            </Link>{" "}
+            with your Pasifika friends and colleagues
+          </Text>
+          <Link href="https://tj8xrxsxqdtxeknk.public.blob.vercel-storage.com/email/pasifika%20flyer-N4j5yIwk22Hb15ESQdYKzHq6LH6K9t.jpg">
+            <Img
+              className="mr-4 inline-block w-1/3 rounded"
+              src="https://tj8xrxsxqdtxeknk.public.blob.vercel-storage.com/email/pasifika%20flyer-N4j5yIwk22Hb15ESQdYKzHq6LH6K9t.jpg"
+            />
+          </Link>
+          <Link href="https://tj8xrxsxqdtxeknk.public.blob.vercel-storage.com/email/pasifika%20postcard-Qqsb3Z578u3UXtzGx05GxFL12L6ZSH.jpg">
+            <Img
+              className="inline-block w-1/3 rounded"
+              src="https://tj8xrxsxqdtxeknk.public.blob.vercel-storage.com/email/pasifika%20postcard-Qqsb3Z578u3UXtzGx05GxFL12L6ZSH.jpg"
+            />
+          </Link>
+          <Text className="mb-0 mt-2 text-xs leading-normal text-stone-500">
+            Feel free to spruce up social posts with either of the full-res
+            graphics made specifically for the event.
+          </Text>
+        </div>
+        <Text>
+          Mahalo to everyone for their kōkua and generous support in making this
+          event happen! Stoked to have collaborated with plenty folks including,
+          but not limited to, friends from Amazon&rsquo;s Pasifika Indigenous@,
+          Hawaiians in Tech, Google Pasifika Peoples Collective, and Apple. E
+          paʻi ka lima kākou!
+        </Text>
+        <Text className="m-0">E mālama,</Text>
+        <Text className="m-0">Taylor Kekai Ho</Text>
+        <Text className="m-0">Hawaiians in Tech</Text>
+      </Layout>
+      <Container className="my-4 text-center text-stone-500">
+        <Link href={`${unsubscribeUrl}`} className="text-xs text-inherit">
+          Unsubscribe
+        </Link>
+        <span className="mx-2">·</span>
+        <Link
+          href="https://hawaiiansintech.org/about?utm_source=confirmation-email-footer"
+          className="text-xs text-inherit"
+        >
+          About
+        </Link>
+        <span className="mx-2">·</span>
+        <Link
+          href="https://hawaiiansintech.org/privacy-policy?utm_source=confirmation-email-footer"
+          className="text-xs text-inherit"
+        >
+          Privacy Policy
+        </Link>
+      </Container>
```

Comment:
Ack, my fault for forgetting on the last one but can you add this below the links. I'm going off memory but as long as it looks mostly good. 👍

```jsx
<Container className="my-4 text-center text-stone-500">
  <Text>200 N Vineyard Blvd A325 - 5374<br />Honolulu, HI 96817</Text>
</Container>
```

Reply:
updated before merge